bedrock/teach/templates/teach/smarton/surveillance.html

```diff
@@ -0,0 +1,344 @@
+{# This Source Code Form is subject to the terms of the Mozilla Public
+ # License, v. 2.0. If a copy of the MPL was not distributed with this
+ # file, You can obtain one at http://mozilla.org/MPL/2.0/. -#}
+
+{% extends "teach/smarton/base.html" %}
+
+{% block page_title %}{{ _('Get smart on government surveillance') }}{% endblock %}
+{% block page_desc %}{{ _('Decide where you think the line should be – and keep others from crossing it. Get smart on government surveillance.') }}{% endblock %}
```

Comment:
String added 20 October: "Decide where you think the line should be – and keep others from crossing it. Get smart on government surveillance."